### ISSUE TITLE
relay integer references

### ISSUE THREAD

**Issue by cryptz2k** [pimoroni/automation-hat#60]
are relays accessible by integer value or by name/word only.

for example:

is:
automationhat.relay[i + 1].on() supported in a for loop? 

or must you spell out the name such as

automationhat.relay.one.toggle()

**Comment by Gadgetoid** [comment]
Yes, there's an explicit case in the deep magic that makes this whacky code work. It should handle addressing relays as a collection.

An example of this might be a good idea in retrospect!
